README.md

```diff
@@ -20,31 +20,39 @@ This Commitizen adapter helps standardize commit messages for freeCodeCamp so
 being asked to fix incorrectly formatted commit messages will be less of a
 problem.
 
-# Local Installation
+# Installation
+
+Install as a global adapter for freeCodeCamp repos
 
 ```shell
-# Install general tool globally
-npm install -g commitizen
+# Install the commitizen cli globally
+npm install --global commitizen
 
-# Install freeCodeCamp specific adapter
-git clone https://github.com/freeCodeCamp/cz-freecodecamp.git
+# Install the freeCodeCamp commitizen adapter globally
+npm install --global cz-freecodecamp
 
-# Add freeCodeCamp plugin to be used
-cd cz-freecodecamp
-echo '{ "path": "'$(pwd)'" }' > ~/.czrc
+# Save the adapter into global config
+echo '{ "path": "cz-freecodecamp" }' > ~/.czrc
```

Comment:
This won't work with global installs. The entire path of the global installs is needed.

For me it's `{ "path": "/usr/local/lib/node_modules/cz-conventional-changelog" }`
But this is going to be specific to the user. We need a way to programmatically get the users global npm install path